Sources/Parsing/Builders/OneOfBuilder.swift

```diff
@@ -1,7 +1,21 @@
-/// A custom parameter attribute that constructs a parser that attempts to run a number of parsers,
-/// one after the other, till one succeeds with an output.
+/// A custom parameter attribute that constructs parsers from closures. The constructed parser
+/// runs each parser in the closure, one after another, till one succeeds with an input.
```

Comment:
I think previously this meant "succeed by returning an input"

```suggestion
/// runs each parser in the closure, one after another, till one succeeds with an output.
```

But I can see this also being "succeed to parse an input". So depending on which you want to focus on...